src/test/kotlin/net/atos/zac/app/taken/TakenRESTServiceTest.kt

```diff
@@ -0,0 +1,161 @@
+import net.atos.zac.app.informatieobjecten.converter.RESTInformatieobjectConverter
+import net.atos.zac.app.taken.converter.RESTTaakConverter
+import net.atos.zac.app.taken.converter.RESTTaakHistorieConverter
+import net.atos.zac.app.taken.model.TaakStatus
+import net.atos.zac.app.taken.model.createRESTTaakToekennenGegevens
+import net.atos.zac.authentication.LoggedInUser
+import net.atos.zac.authentication.createLoggedInUser
+import net.atos.zac.event.EventingService
+import net.atos.zac.event.Opcode
+import net.atos.zac.flowable.TaakVariabelenService
+import net.atos.zac.flowable.TakenService
+import net.atos.zac.policy.PolicyService
+import net.atos.zac.policy.output.createTaakRechten
+import net.atos.zac.shared.helper.OpschortenZaakHelper
+import net.atos.zac.signalering.SignaleringenService
+import net.atos.zac.signalering.event.SignaleringEvent
+import net.atos.zac.signalering.model.SignaleringType
+import net.atos.zac.websocket.event.ScreenEvent
+import net.atos.zac.websocket.event.ScreenEventType
+import net.atos.zac.zoeken.IndexeerService
+import net.atos.zac.zoeken.model.index.ZoekObjectType
+import org.flowable.identitylink.api.IdentityLinkInfo
+import org.flowable.task.api.Task
+import org.junit.jupiter.api.Assertions.assertEquals
+
+@MockKExtension.CheckUnnecessaryStub
+class TakenRESTServiceTest : BehaviorSpec() {
+    val drcClientService = mockk<DRCClientService>()
+    val enkelvoudigInformatieObjectUpdateService = mockk<EnkelvoudigInformatieObjectUpdateService>()
+    val eventingService = mockk<EventingService>()
+    val httpSession = mockk<Instance<HttpSession>>()
+    val indexeerService = mockk<IndexeerService>()
+    val loggedInUserInstance = mockk<Instance<LoggedInUser>>()
+    val opschortenZaakHelper = mockk<OpschortenZaakHelper>()
+    val policyService = mockk<PolicyService>()
+    val restInformatieobjectConverter = mockk<RESTInformatieobjectConverter>()
+    val signaleringenService = mockk<SignaleringenService>()
+    val taakVariabelenService = mockk<TaakVariabelenService>()
+    val restTaakConverter = mockk<RESTTaakConverter>()
+    val taakHistorieConverter = mockk<RESTTaakHistorieConverter>()
+    val takenService = mockk<TakenService>()
+    val zgwApiService = mockk<ZGWApiService>()
+    val zrcClientService = mockk<ZRCClientService>()
+
+    // We have to use @InjectMockKs since the class under test uses field injection instead of constructor injection.
+    // This is because WildFly does not properly support constructor injection for JAX-RS REST services.
+    @InjectMockKs
+    lateinit var takenRESTService: TakenRESTService
+
+    override suspend fun beforeTest(testCase: TestCase) {
+        MockKAnnotations.init(this)
+    }
+
+    init {
+        given("a task is not yet assigned") {
+            When("'toekennen' is called") {
+                then(
+                    "the task is assigned to the provided user and group, " +
```

Comment:
You can use the Kotlin [multiline string](https://kotlinlang.org/docs/strings.html#multiline-strings) here.